src/env.js

```diff
@@ -41,7 +41,8 @@ Sk.python2 = {
     bankers_rounding: false,
     python_version: false,
     dunder_next: false,
-    dunder_round: false,
+    dunder_round: false,    
+    dunder_clear: false,
```

Comment:
please call this `list_clear` 😄 dunder methods are all called like this: `__name__`. So this isn't a dunder method. 😄

Reply:
OK.